File: DayTwo/Python/caeser_cipher.py
def cipher(word, shift):
    letters = "abcdefghijklmnopqrstuvwxyz"
    result = ""
    word = word.lower()

    for index in range(len(word)):
        letter = word[index]
        position = letters.index(letter)
        
        if position == -1:
            result += letter
        
        new_position = position + shift
        result += letters[new_position]
        
    return result

word = "hello"
shift = 3
print(cipher(word, shift))

File: DayTwo/JavaScript/CaeserCipher.js
function cipher(word, shift){

    let letters = "abcdefghijklmnopqrstuvwxyz";
    let result = "";
    word = word.toLowerCase()

    for (let index = 0; index < word.length; index++){
    let letter = word.charAt(index);
            
    let position = letters.indexOf(letter);
            
    if (position == -1){
        result += letter;
        }
    else{
    let newPosition = (position + shift) % 26;
    result += letters.charAt(newPosition);
            }
        }
        return result;
    }

console.log(cipher("hello", 3))

